controller/contour/deployment.go

```diff
@@ -55,6 +57,17 @@ const (
 	// contourDeploymentLabel identifies a deployment as a contour deployment,
 	// and the value is the name of the owning contour.
 	contourDeploymentLabel = "contour.operator.projectcontour.io/deployment-contour"
+	// contourDeploymentSvcAcct is the name of the service account
+	// used by the Contour deployment.
+	contourDeploymentSvcAcct = "contour"
```

Comment:
We should use `contourDeploymentSvcAcct` in `ensureRBAC` as well, but that could be a follow-up.